<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Encargado</class>
 <widget class="QWidget" name="Encargado">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>884</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Encargado</string>
  </property>
  <property name="windowIcon">
   <iconset resource="img.qrc">
    <normaloff>:/imagenes/img/Logo.png</normaloff>:/imagenes/img/Logo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#Encargado{
	background-color: white;
}</string>
  </property>
  <widget class="QWidget" name="menuL" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>61</width>
     <height>601</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#menuL{
	background-color: #4F7302;
}</string>
   </property>
   <widget class="QPushButton" name="btnSalir">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>530</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#btnSalir{
	background: none;
	border: none;
	image: url(:/imagenes/img/logOff.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAsigZ">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>60</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#btnAsigZ{
	background: none;
	border: none;
	image: url(:/imagenes/img/asignarZona.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnGestionUsr">
    <property name="geometry">
     <rect>
      <x>-16</x>
      <y>130</y>
      <width>91</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: none;
border: none;
image: url(:/imagenes/img/corteCaja.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="menuH" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>891</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#menuH{
	background-color: #4F7302;
	border: none;
}</string>
   </property>
   <widget class="QLabel" name="relojD">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>10</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;HelvLight&quot;;
color: #F29F05;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStackedWidget" name="paginas">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>831</width>
     <height>541</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="asignarZona">
    <property name="styleSheet">
     <string notr="true">#asignarZona{
	background-color: white;
	image: url(:/imagenes/img/LogoFaded.png);
}</string>
    </property>
    <widget class="QWidget" name="ficha1" native="true">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>60</y>
       <width>331</width>
       <height>421</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#ficha1{
	border-radius: 15px;
	border:3px solid #A9BF04;
	background-color: white;
}
</string>
     </property>
     <widget class="QComboBox" name="cmbZona">
      <property name="geometry">
       <rect>
        <x>111</x>
        <y>225</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#zonaF1{
	background-color: white;
	border-right: 2px solid #D96704;
	border-bottom: 2px solid #D96704;
	font: 10pt &quot;HelvLight&quot;;
}</string>
      </property>
     </widget>
     <widget class="QPushButton" name="usrIcon1">
      <property name="geometry">
       <rect>
        <x>123</x>
        <y>15</y>
        <width>101</width>
        <height>101</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#usrIcon1{
	background: none;
	border: none;
	image: url(:/imagenes/img/usuario.png);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="nombreM">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>159</y>
        <width>281</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 18pt &quot;HelvLight&quot;;
color: black;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>122</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 16pt &quot;HelvLight&quot;;
color: #D96704;
</string>
      </property>
      <property name="text">
       <string>Mesero:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cmbMeseros">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>120</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#zonaF1{
	background-color: white;
	border-right: 2px solid #D96704;
	border-bottom: 2px solid #D96704;
	font: 10pt &quot;HelvLight&quot;;
}</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>52</x>
        <y>230</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 16pt &quot;HelvLight&quot;;
color: #D96704;
</string>
      </property>
      <property name="text">
       <string>Zona:</string>
      </property>
     </widget>
     <widget class="QLabel" name="nombreZ">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>269</y>
        <width>281</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 18pt &quot;HelvLight&quot;;
color: black;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="btnGuardar">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>342</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#btnGuardar{
	background-color: #D96704;
	border: none;
	font: 16pt &quot;HelvLight&quot;;
	color: white;
}

#btnGuardar:Hover{
	background-color: #A9BF04;
}</string>
      </property>
      <property name="text">
       <string>Guardar</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>0</y>
       <width>311</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 20pt &quot;HelvLight&quot;;
color: #D96704;</string>
     </property>
     <property name="text">
      <string>Asignar zonas a meseros</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="pagina2">
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 20pt &quot;HelvLight&quot;;
color: #D96704;</string>
     </property>
     <property name="text">
      <string>Corte de Caja.</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>60</y>
       <width>421</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;HelvLight&quot;;
color: #00007f;</string>
     </property>
     <property name="text">
      <string>° Seleccione un cajero en especifico para la visualización de ingresos. </string>
     </property>
    </widget>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>110</y>
       <width>281</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox{
border:transparent;
background-color: rgb(255, 255, 255);
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;HelvLight&quot;;</string>
      </property>
      <item>
       <property name="text">
        <string>Cajero 1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Cajero 2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Cajero 3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Cajero 4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Cajero 5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Cajero 6</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>10</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	border: none;
	background-color: #4F7302;
	font: 10pt &quot;HelvLight&quot;;
	color: white;
}

QPushButton:Hover{
	background-color: #D96704;
}</string>
      </property>
      <property name="text">
       <string>Realizar corte</string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="usuario">
     <property name="geometry">
      <rect>
       <x>79</x>
       <y>180</y>
       <width>281</width>
       <height>301</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>30</y>
        <width>131</width>
        <height>131</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="img.qrc">:/imagenes/img/usuario.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="nombre">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>180</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 13pt &quot;HelvLight&quot;;
color: #00007f;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>220</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 15pt &quot;HelvLight&quot;;
color: #00007f;</string>
      </property>
      <property name="text">
       <string>USUARIO</string>
      </property>
     </widget>
     <widget class="QLabel" name="zona">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>250</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;HelvLight&quot;;
color: #00007f;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QTableWidget" name="caja1">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>110</y>
       <width>341</width>
       <height>341</height>
      </rect>
     </property>
    </widget>
    <widget class="QFrame" name="totales">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>450</y>
       <width>351</width>
       <height>71</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;HelvLight&quot;;
color: #00007f;</string>
      </property>
      <property name="text">
       <string>Ordenes totales:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;HelvLight&quot;;
color: #00007f;</string>
      </property>
      <property name="text">
       <string>Ingresos totales:</string>
      </property>
     </widget>
     <widget class="QLabel" name="ordenes">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>10</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;HelvLight&quot;;
color: #00007f;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="ingresos">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>40</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;HelvLight&quot;;
color: #00007f;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>20</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	border: none;
	background-color: #4F7302;
	font: 10pt &quot;HelvLight&quot;;
	color: white;
}

QPushButton:Hover{
	background-color: #D96704;
}</string>
      </property>
      <property name="text">
       <string>Guardar</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections/>
</ui>
